docs/CmdLineQueriesResults.md

```diff
@@ -11,3 +11,8 @@ The result of each command-line query may be 0 or 1 as followed:
 | --connectivity | ConnectivityMap | the result of this query is always 0 | None |
 | --semantic_diff | SemanticDiff | both sets of NetworkPolicies are semantically equivalent | the given sets of NetworkPolicies are not semantically equivalent |
 
+## A query will not be executed when:
+1. The config/s type is not relevant for performing the query.
+2. An input config does not include NetworkPolicies.
+3. The input configs are not comparable.
+4. The input configs of equivalence/semantic-diff query are identical.
```

Comment:
Are these cases relevant for only certain queries? which?

Reply:
my intention here is to describe general cases that may occur.
for example by the first case `The config/s type is not relevant for performing the query.` I mean the queries that do not run on Ingress configurations.
2. corresponds to queries with the `if not self.config` check
3. is for 2 configs queries that call the `is_identical_topologies` def
4. describes a case that is relevant only for equivalence and semantic diff as written